suite-native/module-home/src/components/PortfolioGraphHeader.tsx

```diff
@@ -1,23 +1,30 @@
 import React from 'react';
 
-import { format } from 'date-fns';
 import { atom, useAtom } from 'jotai';
+import { format } from 'date-fns';
 
-import { useFormatters } from '@suite-common/formatters';
 import { Box, Text } from '@suite-native/atoms';
-import { Icon, IconName } from '@trezor/icons';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { ExtendedGraphPoint, LineGraphPoint } from '@suite-common/wallet-graph';
+import { useFormatters } from '@suite-common/formatters';
+import { Icon, IconName } from '@trezor/icons';
 
 // use atomic jotai structure for absolute minimum re-renders and maximum performance
 // otherwise graph will be freezing on slower device while point swipe gesture
-const selectedPointAtom = atom<ExtendedGraphPoint>({
+
+const percentageDiff = (a: number, b: number) => {
+    if (a === 0 || b === 0) return 0;
+    return 100 * ((b - a) / ((b + a) / 2));
+};
+
```

Comment:
Please remove exports for most of these atoms, only writeOnly are public interface

Reply:
fixed